tests/suites/1_stateful/08_select_stage/08_00_parquet/08_00_00_basic.sh

```diff
@@ -26,4 +26,9 @@ echo "drop stage if exists s2;" | $MYSQL_CLIENT_CONNECT
 echo "create stage s2 url = '${DATADIR}' FILE_FORMAT = (type = PARQUET);"  | $MYSQL_CLIENT_CONNECT
 echo "select * from @s2;" | $MYSQL_CLIENT_CONNECT
 
+echo '--- file_format'
+echo "drop stage if exists s3;" | $MYSQL_CLIENT_CONNECT
+echo "create stage s3 url = '${DATADIR}' FILE_FORMAT = (type = CSV);"  | $MYSQL_CLIENT_CONNECT
+echo "select * from @s2 (FILE_FORMAT => 'PARQUET');" | $MYSQL_CLIENT_CONNECT
```

Comment:
The file type option is mainly for from URI not stage, right? stage has its type, we no need to constraint them too

Reply:
https://github.com/datafuselabs/databend/issues/9946

Reply:
I see.